x-pack/legacy/plugins/siem/public/pages/case/components/callout/index.test.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { CaseCallOut } from './';
+
+const defaultProps = {
+  title: 'hey title',
+};
+
+describe('CaseCallOut ', () => {
+  it('Renders single message callout', () => {
+    const props = {
+      ...defaultProps,
+      message: 'we have one message',
+    };
+    const wrapper = mount(<CaseCallOut {...props} />);
+    expect(
+      wrapper
+        .find(`[data-test-subj="callout-message"]`)
+        .last()
+        .exists()
+    ).toBeTruthy();
+    expect(
+      wrapper
+        .find(`[data-test-subj="callout-messages"]`)
+        .last()
+        .exists()
+    ).toBeFalsy();
+  });
+  it('Renders multi message callout', () => {
+    const props = {
+      ...defaultProps,
+      messages: [
+        { ...defaultProps, description: <p>{'we have two messages'}</p> },
+        { ...defaultProps, description: <p>{'for real'}</p> },
+      ],
+    };
+    const wrapper = mount(<CaseCallOut {...props} />);
+    expect(
+      wrapper
+        .find(`[data-test-subj="callout-message"]`)
+        .last()
+        .exists()
+    ).toBeFalsy();
+    expect(
+      wrapper
+        .find(`[data-test-subj="callout-messages"]`)
+        .last()
+        .exists()
+    ).toBeTruthy();
+  });
+  it('Dismisses callout', () => {
+    const props = {
+      ...defaultProps,
+      message: 'we have one message',
+    };
+    const wrapper = mount(<CaseCallOut {...props} />);
+    wrapper
+      .find(`[data-test-subj="callout-dismiss"]`)
+      .last()
+      .simulate('click');
+    expect(wrapper).toEqual({});
```

Comment:
I think is better to put a `data-test-subj` to `EuiCallOut` and check for existence.